### DISCUSSION TITLE
Fleet Password Not Synced with SOC User Password?

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#3530] [Unsupported Versions] [answered]
As usually happens with me and my stupidly painful passwords, I mistyped the SOC user password during setup (I guess twice in perfect harmony? Or maybe one of the characters in the password is at issue - anyway...). I SSH'd into the SO installation and changed the password which worked perfectly - I can log into the SOC console without issue. However, upon trying to log into Fleet... it doesn't recognize the password, which I assume is because it did not update the password when the SOC user's  password was changed. Attempting to locate a way to change the Fleet user password via the terminal resulted in how to do it through the GUI, which of course I cannot utilize. Clicking on the forgot password link on the Fleet login page kicks back an error that email is not configured. Stuck fast I am. Searching r/secruityonion and github yielded nothing relevant. Would someone mind tipping me off on how to crawl out of this hole I've dug?

**Comment by weslambert** [comment, marked as answer]
You might consider using `so-user-add` to add a new user to have the same password across these services.

**Reply by ghost** [reply]
Thanks for the response - much appreciated.
